deps/rabbit/src/rabbit_deprecated_features.erl

```diff
@@ -602,3 +622,8 @@ enable_underlying_feature_flag_cb(
         _ ->
             ok
     end.
+
+is_deprecated_feature_in_use(#{callbacks := #{is_feature_used := {CallbackMod, CallbackFun}}}) ->
+    erlang:apply(CallbackMod, CallbackFun, [#{}]);
```

Comment:
The argument is invalid here. You may want to factorize the execution of the callback from `enable_underlying_feature_flag_db/1` into its own function.